src/sharepoint/account/sharepoint-account.controller.js

```diff
@@ -208,10 +206,6 @@ angular
                 }).catch(() => ({
                     userPrincipalName,
                     activated: false
-                }));
-        }
-
-        onTranformItemDone () {
-            this.loaders.search = false;
+                })).finally(() => { this.loaders.search = false; });
```

Comment:
I'm pretty sure it's not working since as before since previously it was at last element loaded now it's at first